packages/searchkit/src/Transporter.ts

```diff
@@ -67,35 +67,33 @@ export class ESTransporter implements Transporter {
       }
 
       if (responses.status >= 500) {
-        console.error(JSON.stringify(responses))
-        throw new Error(
+        console.error(
           'Elasticsearch Internal Error: Check your elasticsearch instance to make sure it can recieve requests.'
         )
+        throw new Error(JSON.stringify(responses))
       } else if (responses.status === 401) {
-        console.error(JSON.stringify(responses))
-        throw new Error(
+        console.error(
           'Cannot connect to Elasticsearch. Check your connection host and auth details (username/password or API Key required). You can also provide a custom Elasticsearch transporter to the API Client. See https://www.searchkit.co/docs/guides/setup-elasticsearch#connecting-with-usernamepassword for more details.'
         )
+        throw new Error(JSON.stringify(responses))
```

Comment:
Could this potentially expose sensitive information into the logs, like apikeys?